resources/features/Sic.feature

```diff
@@ -0,0 +1,226 @@
+Feature: A sic calculation groups the input data by substrings of SIC and ID to determine which subgroup contains the highest amount of employees
+         and assigns a subdivision determined by the division and group.
+
+         The method uses the following substrings:
+         first 2 characters as division,
+         first 3 characters as group,
+         first 4 characters as class,
+         and the whole whole SIC.
+
+    @Happy Path
+    Scenario Outline: Happy Path - Original Happy Path
+        Given input:
+        | ern | lurn | sic07 | employees |
+        | 123 |    1 | 63111 |       504 |
+        | 123 |    4 | 41202 |       844 |
+        | 123 |    6 | 69201 |       188 |
+        | 123 |   10 | 46444 |       301 |
+        | 123 |   13 | 47344 |       300 |
+        | 123 |   14 | 47844 |       900 |
+        | 123 |   15 | 47944 |       950 |
+        | 345 |    2 | 63110 |        46 |
+        | 345 |    7 | 12312 |       100 |
+        | 345 |   11 | 47544 |       500 |
+        | 345 |    9 | 46212 |       300 |
+        | 456 |    8 | 46123 |       500 |
+        | 456 |    3 | 63120 |       540 |
+        | 654 |    5 | 78109 |       517 |
+        | 654 |   12 | 47144 |      1000 |
+        When the Sic method is calculated
+        Then the Sic results table is produced:
+        | ern | lurn | sic07 | class | group | division | employees |
+        | 123 |   15 | 47944 |  4794 |   479 |       47 |       950 |
+        | 345 |   11 | 47544 |  4754 |   475 |       47 |       500 |
+        | 456 |    3 | 63120 |  6312 |   631 |       63 |       540 |
+        | 654 |   12 | 47144 |  4714 |   471 |       47 |      1000 |
+
+        @JVM
+        Examples:
+        | language |
+        | Scala    |
+
+    @Happy Path
+    Scenario Outline: Happy Path - A Results table is which hits all endpoints of the decision tree
```

Comment:
I'm not sure this Happy Path adds any value to the tests overall as it is simply a collection of each of the separate tests below. Consider removing?